### PR TITLE
[PERFREPO-153] Create new reporter with boxplot chart

### PR DESCRIPTION
**Pull request by Holmistr**

### CODE REVIEW

**Comment by Holmistr** [comment]
@pdrozd Please, do not integrate it yet. The whole report is fully functional, you can begin with the review. However, there are some cosmetic CSS bugs which I want to fix (different width of input text on test search execution page, etc.). Also, I wasn't able to run the TestExecutionDAO test today, but I would swear that it was working. I'll look into it. Wiki page about how to use the boxplot chart should be coming in a few days, hopefully tomorrow :)

File: web/src/main/java/org/perfrepo/web/dao/TestExecutionDAO.java [hunk 1]
```diff
@@ -611,10 +777,8 @@ public static TestExecution fetchTest(TestExecution exec) {
     * @return TestExecution with fetched tags
     */
    public static TestExecution fetchTags(TestExecution testExecution) {
-      Collection<TestExecutionTag> cloneTags = new ArrayList<TestExecutionTag>();
-      for (TestExecutionTag interObject : testExecution.getTestExecutionTags()) {
-         cloneTags.add(interObject.cloneWithTag());
-      }
+      Collection<TestExecutionTag> cloneTags = new ArrayList<>();
+      testExecution.getTestExecutionTags().stream().forEach(interObject -> cloneTags.add(interObject));
```
Comment (pdrozd) [inline web/src/main/java/org/perfrepo/web/dao/TestExecutionDAO.java:781]: Missing interObject.cloneWithTag(), is it ok?

File: web/src/main/webapp/resources/css/design.css [hunk 2]
```diff
@@ -6102,9 +6099,9 @@ table.pointDetails td.pointDetailsValue {
 }
 
 input.rf-au-inp {
-    width: 206px !important;
+    width: 206px;
     border: 1px solid #CCCCCC !important;
-    padding: 4px !important;
+    padding: 4px;
```
Comment (pdrozd) [inline web/src/main/webapp/resources/css/design.css]: Both properties (width, padding) without important can be deleted.

File: web/src/main/resources/lang/strings.properties [hunk 3]
```diff
@@ -153,7 +154,7 @@ page.user.oldPasswordDoesntMatch=Old password doesn't match
 page.showParam.errorNoParamId=No parameter ID supplied
 page.showParam.errorParamNotFound=Can''t find parameter with ID {0}
 
-page.search.asteriskTooltip = The field is case insensitive. \nUse an asterisk as a wildcard at the end of the search string to expand the search results.
+page.search.asteriskTooltip = The field is case insensitive. \nUse an asterisk as a wildcard at the end of the search string to expand the search dataPoints.
```
Comment (pdrozd) [inline web/src/main/resources/lang/strings.properties]: I think it is confusing. At search page, we search test/testExecution not data points

File: web/src/main/java/org/perfrepo/web/dao/TestExecutionDAO.java (hunk 1 above)
Comment (Holmistr) [inline web/src/main/java/org/perfrepo/web/dao/TestExecutionDAO.java:781]: This should not be a problem, default fetching strategy for @OneToOne and @ManyToOne relationships is EAGER, therefore no clone is necessary.

File: web/src/main/webapp/resources/css/design.css (hunk 2 above)
Comment (Holmistr) [inline web/src/main/webapp/resources/css/design.css]: I don't think so. Well, it depends how you use it. If you don't provide any other class like input-xlarge, than there will be a default width. I'm not sure if there is such case anywhere in the application, therefore I prefer not removing it.

File: web/src/main/resources/lang/strings.properties (hunk 3 above)
Comment (Holmistr) [inline web/src/main/resources/lang/strings.properties]: I have no idea why I changed it, right now "results" looks much better. Sorry about that, probably hangover :)

**Comment by Holmistr** [comment]
Rebased, repushed a also fixed https://issues.jboss.org/browse/PERFREPO-188

**Pull request merged**